.github/workflows/main.yml

```diff
@@ -0,0 +1,52 @@
+name: kdump-utils tests
+
+on: pull_request
+
+jobs:
+  format-check:
+    runs-on: ubuntu-latest
+    container: docker.io/fedora:latest
+    steps:
+      - uses: actions/checkout@v4
+      - run: dnf install shfmt -y
+      - run: shfmt -d *.sh kdumpctl mk*dumprd
```

Comment:
`dracut` also uses option `-s` for their tests. Shall we use that as well?
In addition we need to add the new `dracut/`, `tools/` and `tests/` directories.

Reply:
I've used `-s` and added the new directories now, thanks!